django_app/tests/test_migrations.py

```diff
@@ -36,3 +36,44 @@ def test_0012_alter_file_status(migrator):
 
     # Cleanup:
     migrator.reset()
+
+
+@pytest.mark.django_db()
+def test_0019_remove_chatmessage_source_files_textchunk_and_more(migrator):
```

Comment:
World's tiniest nitpick.

```suggestion
def test_0020_remove_chatmessage_source_files_textchunk_and_more(migrator):
```